lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -279,9 +279,15 @@ function _dosomething_campaign_closed_page_access($node) {
  * Determines whether a user has access to the active page callback.
  */
 function _dosomething_campaign_active_page_access($node) {
-  // Staff only.
-  if (!dosomething_user_is_staff()) { return FALSE; }
+  global $user;
+  // Grant access if you're  staff OR you're an org friend:
+  $has_access = dosomething_user_is_staff() || in_array('org friend', $user->roles);
```

Comment:
So, an org friend is definitely not staff. (Staff can see all user info).

The only permission the org friend currently has is to `view any unpublished content`. What do you think about simplifying the query to check for that permission, instead of checking for both Staff OR Org friend?